source/index.html.erb

```diff
@@ -13,119 +13,133 @@
+          </div>
+          <div class="column-half">
+            <img class="content-section__image" src="/pay-product-page/images/product-page_images_user-screen.svg" alt="A user-facing payment page with entry fields for card details and a payment summary. A helpful inline error message responds to user input." />
+          </div>
+      </div>
+    </div>
+    <div class="content-section content-section--with-top-border">
+      <div class="grid-row">
+          <div class="column-half">
+            <h2 class="content-section__title heading-medium">Teams building services will get:</h2>
+            <ul class="list list-bullet">
+              <li>clear documentation on how to integrate with GOV.UK Pay</li>
+              <li>a modern RESTful API for payment initiation and administration</li>
+              <li>a test environment to prototype and test with your service</li>
+            </ul>
+            <p><a href="https://gds-payments.gelato.io/docs/versions/1.0.0">See how GOV.UK Pay works in our documentation</a></p>
+          </div>
+          <div class="column-half">
+            <img class="content-section__image" src="/pay-product-page/images/product-page_images_docs-screen.svg" alt="The online resource of GOV.UK Pay&rsquo;s documentation." />
+          </div>
+      </div>
+    </div>
+    <div class="content-section content-section--with-top-border">
+      <div class="grid-row">
+          <div class="column-half">
+            <h2 class="content-section__title heading-medium">Find out more about GOV.UK Pay</h2>
+            <p><a href="/contact.html">Contact the team</a> to ask a question. Follow our <a href="/getstarted.html">Guide to getting started</a> to create an account.</p>
+            <p>GOV.UK Pay is built and supported 24/7 by a full time team at the Government Digital Service. The service manager is Till Wirth. </p>
+          </div>
+          <div class="column-half">
+            <img class="content-section__image" src="/pay-product-page/images/pay-team.jpg" alt="A photo of the GOV.UK Pay team at work." />
+          </div>
+      </div>
+    </div>
+    <div class="content-section content-section--with-top-border">
+      <div class="grid-row">
+          <div class="column-two-thirds">
+            <h2 class="content-section__title heading-medium">GOV.UK Pay is already working with:</h2>
+            <img class="content-section__image" src="/pay-product-page/images/org-logos-pay.jpg" alt="Logos of Ministry of Justice, Home Office, The Insolvency Service and Office of the Public Guardian" />
+            <div class="panel panel-border-wide">
+              <p class="">&ldquo;I&rsquo;ve just had the first view of [GOV.UK Pay] integrated with our service. It is a really good product. We have integrated it in just 2 weeks which is astonishing and really speaks volumes about [how] well you have set the service up&hellip;&rdquo;</p>
```

Comment:
More characters that can just be included directly rather than as entity references.